Makefile

```diff
@@ -48,13 +48,20 @@ validate-vendor:
 validate-generated-files:
 	./hack/validate-generated-files
 
-validate-all: test lint validate-vendor validate-generated-files
+test-with-cni:
+	./hack/cni-setup
+	./hack/test
```

Comment:
test is containerized. I don't think setting up cni on the host makes sense here.

Reply:
Ohh I thought `--privileged` container will be host setting for network.

Reply:
@AkihiroSuda running CNI inside docker container looks not straightforward. Shall I build buildkit and all move its dependencies on host?

Or if you have any other idea, please suggest.

Reply:
Why do you think it is not straightforward?

Reply:
Creating bridge like `cni0` inside container. I was not able to do it successfully.

Reply:
What was the error you got?

Reply:
Okay, initially I was trying to mount CNI conf and binaries and run tests. The `cni0` was not getting created.
I tried again, but in septate container with CNI binaries and config and I am able to create the `cni0` using this [example](https://github.com/containernetworking/cni#running-a-docker-container-with-network-namespace-set-up-by-cni-plugins).

I think, I will be able to debug the issue with tests now.